sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_delete_images_async.py

```diff
@@ -18,7 +18,8 @@
     Set the environment variables with your own values before running the sample:
     1) CONTAINERREGISTRY_ENDPOINT - The URL of you Container Registry account
 
-    This sample assumes your registry has at least four repositories, run load_registry() if you don't have.
+    This sample assumes your registry has at least one repositories with more than three images,
```

Comment:
Also -- since the sample starts with a call to `load_registry()`, is this comment still necessary? (This applies to each of the samples)
```suggestion
    This sample assumes your registry has at least one repository with more than three images,
```